client/session/from_environment.go

```diff
@@ -36,40 +36,46 @@ const (
 	EnvSpaceliftAPIGitHubToken = "SPACELIFT_API_GITHUB_TOKEN" // #nosec G101
 )
 
+var (
+	ErrEnvSpaceliftAPIKeyID       = fmt.Errorf("%s missing from the environment", EnvSpaceliftAPIKeyID)
+	ErrEnvSpaceliftAPIKeySecret   = fmt.Errorf("%s missing from the environment", EnvSpaceliftAPIKeySecret)
+	ErrEnvSpaceliftAPIKeyEndpoint = fmt.Errorf("%s missing from the environment", EnvSpaceliftAPIKeyEndpoint)
+)
+
 // FromEnvironment creates a Spacelift session from the environment.
 func FromEnvironment(ctx context.Context, client *http.Client) func(func(string) (string, bool)) (Session, error) {
 	return func(lookup func(string) (string, bool)) (Session, error) {
 		if lookup == nil {
 			lookup = os.LookupEnv
 		}
 
-		if token, ok := lookup(EnvSpaceliftAPIToken); ok {
+		if token, ok := lookup(EnvSpaceliftAPIToken); ok && token != "" {
```

Comment:
Just want to make sure I understand the mechanics here - is the intended usage something like:

```bash
SPACELIFT_API_TOKEN="" spacectl ...
```

Reply:
Yeah that is my plan. 

The other option would be to have a variable that allows specifying which credential type to be used regardless of which credential env vars exist, but I think the current approach is easier and more clear.